specification/sdk-tracing.md

```diff
@@ -260,6 +260,40 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
```

Comment:
Not sure if it is a good idea to have a dedicated processor reporting only open spans. This more or less implies that an exporter has to be able to handle more parallel streams or several exporters run in parallel even if they have the same target.
I would prefer an processor which exports open and finished spans. What we need is a clear indication if a span is finished or not.

Reply:
In fact, it might be worth considering if adding an option `exportActiveSpans` to the `BatchSpanProcessor` would be better than a dedicated span processor. I think an advantage would be reduced code duplication in addition to the better synchronization @Flarna mentions. The only disadvantage would be that this may add quite some overhead if the option is not used.

Reply:
> What we need is a clear indication if a span is finished or not.

👍  This is absolutely required.

Reply:
I agree, this should make exporting easier. I'll add this as an option.

Reply:
I agree, this would make exporting easier. I'll add an option to have the span watcher processor also export ended spans for now. We can discuss merging it into the `BatchSpanProcessor` with the community later on.